pipelines/utilities/coredns.sh

```
@@ -0,0 +1,3 @@
+scale_up_coredns(){
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)